charts/kubescape-operator/templates/autoupdater/cronjob.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if eq .Values.capabilities.autoUpgrading "enable" }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ .Values.helmReleaseUpgrader.name }}
+  namespace: {{ .Values.ksNamespace }}
+  annotations:
+    "helm.sh/resource-policy": keep
+spec:
+  schedule: {{ .Values.helmReleaseUpgrader.schedule | quote }}
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: {{ .Values.helmReleaseUpgrader.name }}
+          securityContext:
+            allowPriviledgeEscalation: false
+            runAsNonRoot: true
+            runAsUser: 1000
```

Comment:
thought: I was not able to use `readonlyRootFilesystem: true` here, because Helm requires some ephemeral files to write its `.config` and `.cache` stuff.

Reply:
use emptyDir{} mounted where you need to write

Reply:
Today I learned. Convenient stuff. Thanks!

Reply:
Tested and works.